src/ReservationItem.php

```diff
@@ -786,8 +787,11 @@ public static function cronReservation($task = null)
                 ],
                 'WHERE'     => [
                     'glpi_reservationitems.entities_id' => $entity,
-                    new QueryExpression('(UNIX_TIMESTAMP(' . $DB->quoteName('glpi_reservations.end') . ') - ' . $secs . ') < UNIX_TIMESTAMP()'),
-                    'glpi_reservations.begin'  => ['<', new QueryExpression('NOW()')],
+                    new QueryExpression(
+                        QueryFunction::unixTimestamp('glpi_reservations.end') . ' - ' . $secs .
+                            ' < ' . QueryFunction::unixTimestamp(QueryFunction::now())
```

Comment:
We should make the `$expression` parameter optional for `QueryFunction::unixTimestamp()`, in order to simplify its usage.

```suggestion
                            ' < ' . QueryFunction::unixTimestamp()
```